hot/update-panel-elems.js

```diff
@@ -0,0 +1,22 @@
+module.exports = function updatePanelElems(elemName, updateFn) {
+  const elems = document.querySelectorAll(`body /deep/ ${elemName}`);
```

Comment:
`/deep/` has been deprecated for a while now and I think this will break entirely with the upgrade in https://github.com/mixpanel/panel/pull/18. Not sure but https://developers.google.com/web/fundamentals/web-components/shadowdom#findall may be the only path forward here ATM.